src/state/types.ts

```diff
@@ -138,6 +138,8 @@ export type WhiteboardSlideInstance = {
   addElements(elements: Array<Element>): string[];
   /** Remove the element */
   removeElement(elementId: string): void;
+  /** Remove the elements by their IDs */
+  removeElements(elementIds: string[]): void;
```

Comment:
I think a singular version: `removeElement` is only used in command+X shortcut currently. I don't think that we have to change the shortcut in this PR, need to do it later, but just a note to have in mind.

Reply:
You mean remove the singular version once all places are migrated to the other one?

Reply:
yes, correct, I don't think we need singular version of delete and some others at the end.